backend/db/insert.js

```diff
@@ -0,0 +1,18 @@
+const db = require('./config')
+
+async function insertData() {
```

Comment:
Aproveitando para as próximas, esta a forma tradicional de declarar funções em JS que traz alguns problemas. JS é uma linguagem que permite uma imensa agilidade de desenvolvimento, mas por sua 1ª versão ter sido criada em 10 dias, ela tem inúmeros problemas que levam facilmente a bugs no software. Por isso, é mais do que essencial seguir as melhores práticas com JS.

Uma dessas práticas é declarar tudo como `const`, até que vc perceba que precisa que algo seja variável (e usar `let` no caso, nunca `var`).

O problema de declarar funções da forma tradicional é que funções em JS são como variáveis (isso é coisa de programação funcional). Veja o exemplo:

```javascript
function minhaFuncao(){
    console.log('Chamando minhaFuncao');
}

minhaFuncao(); 

//Substitui a implementação da minhaFuncao por outra qualquer
//(minhaFuncao é apenas uma variável que armazena uma função)
minhaFuncao = function (){ console.log('Hackeando a minhaFuncao, trocando o código dela'); }

minhaFuncao(); 
```

Para evitar tais problemas, sempre use arrow functions quando possível, declarando a função como `const`:

```javascript
//se for async: const minhaFuncao = async () => console.log('Chamando minhaFuncao');
const minhaFuncao = () => console.log('Chamando minhaFuncao');

minhaFuncao(); 

//ERRO: Tentando substituir a implementação da minhaFuncao por outra qualquer, não vai funcionar
minhaFuncao = function (){ console.log('Hackeando a minhaFuncao, trocando o código dela'); }

minhaFuncao(); 
```